### ISSUE TITLE
URL permissions cannot be set for third party web services

### ISSUE THREAD

**Issue by soham2008xyz** [toggl/toggl-button#715]
Please follow this guide when creating a new issue. This speeds up the process of replicating the issue and finding a solution.

### Environment details

* Operation system: Ubuntu 16.04
* Extension version: 1.0.16

### Expected behavior
The Toggl extension options screen should have a list of available third-party web sites supported by Toggl button.

### Actual behavior
The list of supported third-party apps is empty.

### Steps to reproduce the behavior
Go to Toggl button extension settings from Chrome extensions page.

### Additional details
This does not work on the Windows version of Chrome as well.

* screenshots or screencapture
![screenshot from 2016-11-28 11 34 32](https://cloud.githubusercontent.com/assets/7334295/20657798/d039e956-b55e-11e6-946b-6e2248009b31.png)

**Comment by IndrekV** [comment]
I was able to replicate the issue. For me the issue happened when I was not logged in. I closed the settings and logged in from the popup and then reopened the settings and everything was fine. List was filled with urls of different tools.

**Comment by soham2008xyz** [comment]
I can confirm that the bug persists even after logging out and logging in from the popup. For good measure, I uninstalled and reinstalled the extension as well, but to no avail.

Screenshot:
![screenshot from 2016-11-29 22 53 37](https://cloud.githubusercontent.com/assets/7334295/20720934/1c4b5054-b687-11e6-9a6f-9b93ed117927.png)

**Comment by IndrekV** [comment]
@soham2008xyz 
Can you please check if the developer console shows any errors.

It should open when clicking on the `background page` link  in the extenstions listing page.

![screen shot 2016-11-30 at 09 45 58](https://cloud.githubusercontent.com/assets/842229/20743979/fa99d06a-b6e1-11e6-9649-662660636b46.png)

When the developer console opens please select `Console` tab from the top or maybe it's already opened in the bottom as on the screens shot. 

![screen shot 2016-11-30 at 09 47 02](https://cloud.githubusercontent.com/assets/842229/20744029/3adb9d98-b6e2-11e6-940a-502707e0d816.png)

Then refresh the developer console (Ctrl+R or cmd+r) and see if any errors appear. If errors appear send them to me so I can try to replicate and fix.

**Comment by soham2008xyz** [comment]
I opened the developer console, it showed a couple of  errors (probably unrelated though, I'm on a flaky connection here). Attaching screenshot:
![screenshot from 2016-11-30 14 32 31](https://cloud.githubusercontent.com/assets/7334295/20746102/63c88e50-b70a-11e6-82c1-8ef7a7ef924c.png)

I refreshed the dev console (Ctrl+R) and the errors went away. I went back and opened the Toggl extensio options page, and the bug persists. However, the dev console output is still empty. Screenshot of extension options page:
![screenshot from 2016-11-30 14 34 28](https://cloud.githubusercontent.com/assets/7334295/20746152/9c902b44-b70a-11e6-8eed-8c74c6adf237.png)
